.pre-commit-config.yaml

```
@@ -0,0 +1,11 @@
+files: "sgeop\/|notebooks\/"
```

```suggestion
files: "sgeop\/"
```

I don't expect notebooks folder to be in this repo. We may have notebooks in docs but let's add those once docs land.